test/powershell/Modules/Microsoft.PowerShell.Core/Remove-Module.Tests.ps1

```diff
@@ -1,19 +1,406 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
+Describe "Remove-Module -Name | -FullyQualifiedName | -ModuleInfo" -Tags "CI" {
```

Comment:
Please add new line after the license text.

Reply:
Done :)